docs/source/technical_tutorials/access/accessing_data.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I suggest either merging this cell with the code cell above or moving this code cell before the above markdown cell.</p><p> </p><p>Does this make sense to use the requests library? the getLocalPath() functionality doesn't seem to have gone away. If we're looking to simplify this action, maybe just getData()?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Agree that it should be moved up above the markdown or merged with code above</p>